seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/sink/writer/DorisSinkWriter.java

```diff
@@ -222,9 +220,11 @@ private void checkDone() {
         }
     }
 
-    private void checkLoadException() {
+    private void checkLoadExceptionAndResetThread() {
         if (loadException != null) {
             throw new RuntimeException("error while loading data.", loadException);
+        } else {
+            executorThread = Thread.currentThread();
```

Comment:
Before this PR, the `executorThread` value not right. So that stream load check error thread can not interupt write thread normally.